helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml

```diff
@@ -0,0 +1,78 @@
+{{ if and (eq .Values.provider "jaeger") (eq .Values.jaeger.template "all-in-one") }}
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "jaeger"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "tracing.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  strategy: allInOne
+
+  allInOne:
+    image: {{ .Values.jaeger.hub }}/all-in-one:{{ .Values.jaeger.tag }}
```

Comment:
The upstream image is `all-in-one` but downstream is `jaeger-all-in-one`, so not sure how best to handle this?

Reply:
I'll modify the patch-charts.sh script to account for that.  Other option is to create an `image` entry for each image (collector, agent, query, all-in-one), but we'd still have to modify the default values (although, things like that are handy when testing).

Reply:
addressed in patch-jaeger.sh.  for non-community builds, this will be jaeger-all-in-one.